rbi/annotations/activesupport.rbi

```diff
@@ -46,3 +46,6 @@ class Object
   sig { returns(T::Boolean) }
   def present?; end
 end
+
+class ActiveSupport::TimeWithZone < Time
+end
```

Comment:
I think representing it as a mixin is slightly more accurate and you can satisfy our CI using `@shim`, similar to https://github.com/Shopify/rbi-central/blob/cd36dd5bc194097e703c384d8e7b94370be45df8/rbi/annotations/activerecord.rbi#L9

Reply:
Perf, will change